docs/source/getting_started/index.md

```diff
@@ -118,6 +118,7 @@ async def run_main():
         model=os.environ["INFERENCE_MODEL"],
         instructions="You are a helpful assistant",
         tools=[{"type": "memory"}],  # enable Memory aka RAG
+        enable_session_persistence=True,
```

Comment:
Fixes:

<img width="1061" alt="Screenshot 2024-12-06 at 19 19 18" src="https://github.com/user-attachments/assets/63f88b80-a709-4c82-8777-44c8d81effef">

Parameter is now required:

https://github.com/meta-llama/llama-stack-client-python/blob/eb3d21932d22b863e5687eed2ce26e713ab46682/src/llama_stack_client/types/shared/agent_config.py#L28